scripts/docker/supervisor.conf

```diff
@@ -10,3 +10,12 @@ redirect_stderr=true
 stdout_logfile=logs/docker.log
 stdout_logfile_maxbytes=1MB
 stopsignal=KILL
+
+[program:receipt-verifier]
+command=gunicorn --log-level=DEBUG -c wsgi/receiptverify.py -b 0.0.0.0:9000 --debug verify:application
```

Comment:
For other apps in Docker we've been avoiding the use of default ports so how about making this something non-default (9001 or whatever) ? I believe 9000 is a default used by Pyramid which probably doesn't matter much but there could be other frameworks that use it.